frontend/src/components/common/Switch/Switch.stories.tsx

```diff
@@ -0,0 +1,238 @@
+import { useState } from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import Switch from './Switch';
+
+/**
+ * 공용 Switch 컴포넌트
+ */
+const meta: Meta<typeof Switch> = {
+  title: 'Common/Switch',
+  component: Switch,
+  tags: ['autodocs'],
+  render: (args) => {
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    const [checked, setChecked] = useState(false);
```

Comment:
왜 `eslint-disable`를 썼는지 궁금해서 주석 지워봤는데, 이런 난감한 문제가 있었구나?

꼼수에 가까운 방법이기는 하지만, 이러한 방법을 써서 린터 오류를 막을 수 있어:
- https://github.com/storybookjs/storybook/blob/f0febc6a5060b43ab711a0006bcd8e8557092bdf/code/renderers/react/template/stories/decorators.stories.tsx#L43

```suggestion
  render: function Function(args) {
    const [checked, setChecked] = useState(false);
```

지금 문제인 점은 `useState` 가 컴포넌트, 커스텀 훅 둘 중 하나에 들어있지 않아 발생하는데, 이 꼼수의 경우 함수명을 `Function` 과 같은 대문자로 해서 린터 에러를 피해가는 방법이야. 몇몇 개발자가 사용하는 것 같은데 별로다 싶으면 적용하지 않아도 괜찮아

아니면, `.stories.tsx`에 한해 규칙을 비활성화하는 걸 고려해 볼까?

Reply:
오 이런게 있구만 반영했어요! 규칙 비활성화는 굳이 안해도 될 것 같아유